src/encoded/static/components/cart/manager.js

```diff
@@ -541,7 +550,7 @@ DeleteCartButtonComponent.propTypes = {
     setInProgress: PropTypes.func.isRequired,
     /** Function to call to update the cart manager page */
     updateCartManager: PropTypes.func.isRequired,
-    /** System function for xhr requests */
+    /** System function for XHR requests */
```

Comment:
XHR refers to XMLHttpRequest. Fetch API is a different architecture.

Reply:
Yeah, that’s true.